ci/pipelines/prometheus-pushgateway.yml

```diff
@@ -37,6 +72,33 @@ resource_types:
       tag: latest
 
 jobs:
+  - name: push-adot-to-deploy-ecr
+    plan:
+      - get: adot-ecr-registry-staging
+        params:
+          format: oci
+        trigger: true
+      - load_var: application_image_tag
+        file: adot-ecr-registry-staging/tag
+      - get: pay-ci
+      - task: parse-staging-release-tag
+        file: pay-ci/ci/tasks/parse-staging-release-tag.yml
+        input_mapping:
+          ecr-repo: adot-ecr-registry-staging
+      - load_var: parse_staging_release_tag
+        file: parse-staging-release-tag/tag
+      - task: send-job-release-info-to-hosted-graphite
+        file: pay-ci/ci/tasks/send-job-release-info-to-hosted-graphite.yml
+        params:
+          BUILD_PIPELINE_NAME: deploy-to-staging
+          BUILD_JOB_NAME: push-adot-to-production-ecr
+          RELEASE_NUMBER: ((.:parse_staging_release_tag))
+          HOSTED_GRAPHITE_API_TOKEN: ((HOSTED_GRAPHITE_API_KEY))
+      - put: adot-ecr-registry-deploy
+        params:
```

Comment:
Can you add the skip download option to this like this: https://github.com/alphagov/pay-ci/blob/master/ci/pipelines/deploy-to-test.yml#L1202-L1203

By default after the push it will then pull the image again which takes time and resources, and isn't useful :( (very annoying behaviour!)